jest.config.js

```diff
@@ -55,6 +55,7 @@ module.exports = {
   moduleFileExtensions: ["js", "json", "es6", "ts", "tsx"],
   modulePathIgnorePatterns: ["/tmp/", "/node_modules/", "/.module-cache/"],
   moduleNameMapper: {
+    "^data-service$": "<rootDir>/packages/data-service/src/index.js",
```

Comment:
I would like to understand why do we need that change. On bills PR, I was actually able to remove the code he added to use the data-service in jest.

Reply:
might be obsolete, dima fixed something in the data-service types 🤔